resharper/resharper-unity/src/Application/UI/Options/VisualStudio/ReSharperOptionsPage.cs

```diff
@@ -64,6 +64,11 @@ public class ReSharperOptionsPage : BeSimpleOptionsPage
             AddBoolOption((UnitySettings s) => s.IsAssetIndexingEnabled,
                 "Parse text based asset files for implicit script usages (requires re-opening solution)");
 
+            
+            AddHeader("Shaders");
+            AddBoolOption((UnitySettings s) => s.SuppressShaderErrorHighlighting,
+                "Suppress resolve errors of unqualified names in HLSL");
```

Comment:
Why "HLSL" here, when we're currently only checking `CGPROGRAM` and `CGINCLUDE`?

Reply:
Not only CGPROGRAM and CGINCLUDE, we also support HLSLPROGRAM/HLSLINCLUDE, .hlsl files, .cg files. All that files use HLSL language